# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/browsers/windows/iframe-cross-origin-print.sub.html.ini
# Commit: c2741585119b
# Full Hash: c2741585119ba8536918b94429296479c0865c25
# Author: Hiroyuki Ikezoe <hikezoe.birchill@mozilla.com>
# Date: 2021-07-27 09:45:43
# Regressor Bug: 1695806
# File Overlap Count: 1
# Description:
#   Bug 1695806 - Defer sending a snapshot reqeust for remote process iframe until cloning the iframe document finished. r=mattwoodrow,nika
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D119950
# ==============================================================================

diff -r 3c5186df3677 -r c2741585119b testing/web-platform/meta/html/browsers/windows/iframe-cross-origin-print.sub.html.ini
--- a/testing/web-platform/meta/html/browsers/windows/iframe-cross-origin-print.sub.html.ini	Tue Jul 27 00:47:44 2021 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,9 +0,0 @@
-[iframe-cross-origin-print.sub.html]
-  bug:
-    if fission: https://bugzilla.mozilla.org/show_bug.cgi?id=1695806
-  expected:
-    if fission and (os == "linux") and not debug and not webrender: [FAIL, PASS]
-    if fission and (os == "linux") and not debug and webrender: [PASS, FAIL]
-    if fission and (os == "linux") and debug: [FAIL, PASS]
-    if fission and (os == "win") and not debug: [PASS, FAIL]
-    if fission and (os == "win") and debug: [FAIL, PASS]
